src/components/About.js

```diff
@@ -0,0 +1,165 @@
+                ipsa accusantium doloremque rem laudantium totam aperiam.
+              </p>
+              <ul className="list-unstyled mb-0">
+                <a href="#!" className="p-2 fa-lg">
+                  <FaFacebook className="blue-text" />
+                </a>
+                <a href="#!" className="p-2 fa-lg">
+                  <FaInstagram className="blue-text" />
+                </a>
+                <a href="#!" className="p-2 fa-lg">
+                  <FaTwitter className="blue-text" />
+                </a>
+              </ul>
+            </Col>
+            <Col lg="3" md="6" className="mb-lg-0 mb-5">
+              <Avatar
+                tag="img"
+                src="https://mdbootstrap.com/img/Photos/Avatars/img%20(30).jpg"
+                className="rounded-circle z-depth-1 img-fluid"
+                alt="Sample avatar"
+              />
+              <h5 className="font-weight-bold mt-4 mb-3">Maria Smith</h5>
+              <p className="text-uppercase blue-text">Manager</p>
+              <p className="grey-text">
+                Excepteur sint occaecat cupidatat non proident, sunt in culpa
+                qui officia deserunt mollit anim est fugiat nulla id eu laborum.
+              </p>
+              <ul className="list-unstyled mb-0">
+                <a href="#!" className="p-2 fa-lg">
+                  <FaFacebook className="blue-text" />
+                </a>
+                <a href="#!" className="p-2 fa-lg">
+                  <FaInstagram className="blue-text" />
+                </a>
+                <a href="#!" className="p-2 fa-lg">
+                  <FaTwitter className="blue-text" />
+                </a>
+              </ul>
+            </Col>
+            <Col lg="3" md="6" className="mb-lg-0 mb-5">
+              <Avatar
+                tag="img"
+                src="https://mdbootstrap.com/img/Photos/Avatars/img%20(32).jpg"
+                className="rounded-circle z-depth-1 img-fluid"
+                alt="Sample avatar"
+              />
+              <h5 className="font-weight-bold mt-4 mb-3">Tom Adams</h5>
+              <p className="text-uppercase blue-text">Receptionist</p>
+              <p className="grey-text">
+                Perspiciatis repellendus ad odit consequuntur, eveniet earum
+                nisi qui consectetur totam officia voluptates perferendis
+                voluptatibus aut.
+              </p>
+              <ul className="list-unstyled mb-0">
+                <a href="#!" className="p-2 fa-lg">
+                  <FaFacebook className="blue-text" />
+                </a>
+                <a href="#!" className="p-2 fa-lg">
+                  <FaInstagram className="blue-text" />
+                </a>
+                <a href="#!" className="p-2 fa-lg">
+                  <FaTwitter className="blue-text" />
+                </a>
+              </ul>
+            </Col>
+            </Row>
+          </Card.Body>
+          </Card>
+  </div>
```

Comment:
The indentation here is misaligned. This makes the file harder to read. I suggest using Prettier (or similar) to format your code.